app/components/restructured_work_history/job_component.html.erb

```diff
@@ -14,6 +14,14 @@
 
     <% if @work_experience.relevant_skills %>
       <p class="govuk-body-s govuk-!-margin-bottom-1">This role used skills relevant to teaching</p>
+    <% elsif @work_experience.relevant_skills.nil? && @editable %>
+      <%= govuk_inset_text(classes: 'govuk-error-summary__list app-inset-text--error') do %>
+        <%= govuk_link_to(
+          'Select if this role used skills relevant to teaching',
+          candidate_interface_edit_restructured_work_history_path(@work_experience.id),
+          class: 'govuk-!-margin-right-2',
```

Comment:
Was there a need to set this margin right?

Reply:
Nope, not sure where I copied that from but it's completely redundant. Deleted.